lib/screens/non_premium/send2.dart

```diff
@@ -247,34 +248,39 @@ class Send2State extends State<Send2> {
       child: Scaffold(
         appBar: AppBar(
           elevation: 0,
-          title: const Text('Bitcoin Address'),
+          title: const Text(
```

Comment:
Note that the apptextstyle for the appbar title is set globally at https://github.com/CalebCouch/orangeme/blob/master/lib/styles/theme.dart

It is globally set to heading4 but can be overridden locally like you've done here, however, when I check this screen in Design v2.5 I see that it calls for a heading4 on this screen, I believe only "Messages" and "Wallet" call for heading3